src/main/java/com/amazon/opendistroforelasticsearch/sql/utils/SQLFunctions.java

```diff
@@ -509,6 +531,16 @@ private static String getPropertyOrValue(SQLExpr expr) {
         }
     }
 
+    private static String getPropertyOrValue(String expr) {
+        if (expr.startsWith("'") && expr.endsWith("'")) {
```

Comment:
In SQL standard, is double quote also valid string literal? Do we have the original expr to pass in here? I think check on AST expr is more reliable.

Reply:
The quoted string literal is parsed to SQLCharExpr regardless of being single quoted or double quoted.